libcloud/compute/types.py

```diff
@@ -236,15 +236,27 @@ class StorageVolumeState(object):
     """
     Standard states of a StorageVolume
     """
+    AVAILABLE = "available"
```

Comment:
Sorry, for a late comment, I missed this in the previous pass.

For consistency, it would be great if you can change "INUSE" to "IN_USE" and "BACKUP" to "SNAPSHOTING" (this way it's a verb and it's consistent with other states).

Thanks!